src/hba.c

```diff
@@ -31,8 +31,9 @@ enum RuleType {
 	RULE_HOSTNOSSL,
 };
 
-#define NAME_ALL        1
-#define NAME_SAMEUSER   2
+#define NAME_ALL                1
+#define NAME_SAMEUSER           2
+#define NAME_REPLICATION        4
```

Comment:
We should comment here that this is a bit mask. It wasn't clear from the first two values, but with the third one it becomes relevant.